package.json

```diff
@@ -19,7 +19,10 @@
     "lint": "eslint --ext js,jsx,md ./",
     "test:git-history": "commitlint --from origin/master --to HEAD",
     "test:lockfile": "node lockFileLint.js",
-    "posttest": "npm run test:lockfile"
+    "posttest": "npm run test:lockfile",
+    "lerna:version": "lerna version",
```

Comment:
Would you like to document this in a release.md file similar to this https://github.com/americanexpress/one-app-cli/blob/master/CONTRIBUTING.md#creating-a-one-app-cli-new-release

Reply:
Yeah, we should add a CONTRIBUTING.md file in a subsequent PR.  I'll add an issue for it.

Reply:
#51